charts/rucio-daemons/templates/policy-packages-init.yaml

```diff
@@ -18,6 +18,7 @@ initContainers:
               dnf install --assumeyes git-all
           fi
           dnf install --assumeyes python-pip
+          pip uninstall -y {{ .moduleName }}
```

Comment:
I always recommend using the long form of an option (in this case `--yes`) in scripts because it offers superior self-documentation, but this is a personal preference.